src/Pact/Eval.hs

```diff
@@ -324,7 +324,8 @@ eval' (TModule _tm@(MDModule m) bod i) =
         void $ acquireModuleAdminCapability capMName $ return ()
     -- build/install module from defs
     (g,govM) <- loadModule mangledM bod i g0
-    _ <- computeGas (Left (i,"module")) (GPreWrite (WriteModule (_mName m) (_mCode m)))
+    szVer <- ifExecutionFlagSet' FlagDisablePact45 SizeOfV0 SizeOfV1
```

Comment:
Need a `getSizeOfVersion` to do this compute, way too much copypasta.

Reply:
Sure.